x/marker/keeper/migrations.go

```diff
@@ -18,8 +18,11 @@ func NewMigrator(keeper Keeper) Migrator {
 // Migrate1to2 migrates from version 1 to 2.
 func (m *Migrator) Migrate1to2(ctx sdk.Context) error {
 	ctx.Logger().Info("Migrating Marker Module from Version 1 to 2")
-	if err := v042.MigrateMarkerAddressKeys(ctx, m.keeper.storeKey); err != nil {
+	err := v042.MigrateMarkerAddressKeys(ctx, m.keeper.storeKey)
+	if err != nil {
```

Comment:
#nit not important but isn't the original way more idiomatic to go?

Reply:
You are correct, but I wanted to use the err variable in the next statement.